### PR TITLE
Frontend/#636

### PR DESCRIPTION
**Pull request by Vayras**
## workspace
https://www.loom.com/share/e8cb4cdefbc84d37904664c1d118612a

##person bounties
https://www.loom.com/share/40e893e95b9a4d6fb304db9c2aeb13f4

## bounties page
https://www.loom.com/share/843be30f66f5452b8771f0aee3e0271c

## Describe your changes
consistency amongst filters

## Issue ticket number and link
#636 

## Type of change
bug fixes for user experience

Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [x] I have tested on a mobile device
- [x] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by humansinstitute** [comment]
Are you able to address the failed test? 

image

**Comment by Vayras** [comment]
@humansinstitute yeah! Just working on that won't be long 🚀

File: src/store/main.ts [hunk 1]
```diff
@@ -997,29 +1046,40 @@ export class MainStore {
...
+      queryLimit,
+      params ? queryParams : this.getWantedsSpecWorkspacePrevParams
+    );
 
     try {
       const ps2 = await api.get(query2);
       const ps3: any[] = [];
 
-      if (ps2 && ps2.length) {
+      if (ps2) {
```
Comment (elraphty) [inline src/store/main.ts:1071]: This should be a backend fix

**Comment by elraphty** [comment]
@humansinstitute, I think this should be a backend fix, the filters are sent to the backend.

**Comment by Vayras** [comment]
@elraphty agreed need more data from back-end that would be much better

**Comment by Vayras** [comment]
But lot of current data coming from back-end is very messy so this might come in handy while backend issue is being fixed 🤔

**Comment by Vayras** [comment]
@humansinstitute do you have any follow ups for me on this? Let me know Thanks

**Pull request merged**